requirements/prod.txt

```diff
@@ -1,5 +1,6 @@
 -r compiled.txt
-amo-validator==1.9.7
+#amo-validator==1.9.7
+-e git+https://github.com/magopian/amo-validator@e425576b948f5f967ff7fe00f687fe103b662b21#egg=amo-validator
```

Comment:
Yep! As soon as the other PR on the validator is merged, I can release a new version of the validator, and use it here instead.